### PR TITLE
fix/6884 removing call button and it's corresponding logic

### PR DESCRIPTION
**Pull request by naveeddotio**
## Description
Since we have two hotline numbers, we have to remove call button and it's corresponding logic to make it consistent with android. 

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-6884

## Screenshots
Simulator Screen Shot - Clone 1 of iPhone 12 - 2021-05-06 at 19 56 43

### CODE REVIEW

**Review by whiskey — APPROVED** [review]

File: src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/ExposureSubmissionHotlineViewController.swift [hunk 1]
```diff
@@ -35,7 +35,6 @@ class ExposureSubmissionHotlineViewController: DynamicTableViewController, ENANa
 		setupBackButton()
 		
 		footerView?.primaryButton.accessibilityIdentifier = AccessibilityIdentifiers.ExposureSubmissionHotline.primaryButton
-		footerView?.secondaryButton.accessibilityIdentifier = AccessibilityIdentifiers.ExposureSubmissionHotline.secondaryButton
```
Comment (pascal-brause) [inline src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/ExposureSubmissionHotlineViewController.swift:38]: Can we remove the `AccessibilityIdentifiers.ExposureSubmissionHotline.secondaryButton` also from the strings-file?

File: src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/ExposureSubmissionHotlineViewController.swift [hunk 2]
```diff
@@ -50,25 +49,19 @@ class ExposureSubmissionHotlineViewController: DynamicTableViewController, ENANa
...
 	}
 
 	// MARK: - Private
 	
-	private let onSecondaryButtonTap: () -> Void
+	private let onPrimaryButtonTap: () -> Void
 
 	private lazy var navigationFooterItem: ENANavigationFooterItem = {
 		let item = ENANavigationFooterItem()
-		item.primaryButtonTitle = AppStrings.ExposureSubmissionHotline.callButtonTitle
```
Comment (pascal-brause) [inline src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/ExposureSubmissionHotlineViewController.swift:66]: Can we remove `AppStrings.ExposureSubmissionHotline.callButtonTitle` from the strings-file?

File: src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/ExposureSubmissionHotlineViewController.swift [hunk 3]
```diff
@@ -50,25 +49,19 @@ class ExposureSubmissionHotlineViewController: DynamicTableViewController, ENANa
...
+		item.primaryButtonTitle = AppStrings.ExposureSubmissionHotline.tanInputButtonTitle
 		item.isPrimaryButtonEnabled = true
```
Comment (pascal-brause) [inline src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/ExposureSubmissionHotlineViewController.swift:62]: do we need here additionally `item.isPrimaryButtonHidden = false`?

Comment (naveeddotio) [reply src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/ExposureSubmissionHotlineViewController.swift:62]: @KaiTeuber Do you think we need this?

**Comment by marcussc** [comment]
Tried to check it locally and it doesn't compile :/

Two other remarks; 
- please create the branches according to our branchnaming convention. 
- Create a comment on the Jira ticket which links to this PR

File: src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/ExposureSubmissionHotlineViewController.swift (hunk 1 above)
Comment (naveeddotio) [reply src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/ExposureSubmissionHotlineViewController.swift:38]: No we need this

File: src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/ExposureSubmissionHotlineViewController.swift (hunk 2 above)
Comment (naveeddotio) [reply src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/ExposureSubmissionHotlineViewController.swift:66]: No we need this

**Review by marcussc — APPROVED** [review]

**Review by pascal-brause — APPROVED** [review]
if it runs and it works as expected, my comment about the primaryButton can be ignored

**Pull request merged**